### ISSUE TITLE
Adding M117 / M00 to the TFT Firmware

### ISSUE THREAD

**Issue by Rics3D** [makerbase-mks/MKS-TFT#205]
This is a message specially for the developers at MKS.
To make the MKS TFT Display fully useable you have to add the Feature, that the Display can show M117 Messages. Also a window is needed after reading  a M00 in the gcode. In this window must be a button to start the programm after a M00 command . 

After this changes in the main Firmware, the display is fully usebal

**Comment by makerbase-mks** [comment]
Thanks for your advice,we'll consider it.
You can make a custom design for it if you need the function.

**Comment by Rics3D** [comment]
Hi, I can't.
It's a firmware thing, not a thing to configure in the .txt file.

**Comment by superdbop** [comment]
I second this. We definitely need M0/M1 support (and whatever the equivalent is in non-marlin firmware.) It should include a countdown timer if applicable and a message string + "Continue" button. As for M117, it's just a simple text string on the printing status screen.

**Comment by JosueAlejandro89** [comment]
It will be an awesome future to have the chance to display messages as we can set scripts in the gcode with the slicer and reported to the user to perform actions or notify status.

**Comment by IrishMoss** [comment]
I agree completely.  Without the ability to output messages to the LCD, it limits how much we can interact with the interface.

**Comment by scmanjarrez** [comment]
Completely agree, we are limited so much if we can't interact on error messages like filament sensor or M0 commands.

**Comment by ebayo-mks** [comment]
+1

**Comment by geins** [comment]
+1
Definitly needed, otherwise, the TFT stays just a gimmick and no professional replacement

**Comment by nuansed** [comment]
+1 Would love this feature, especially in conjunction with Octoprint!

**Comment by DerJenigeDotInc** [comment]
+1
On my Sidewinder X1 it starts beeping when heating up the bed (from panel or from gcode) when i just send once an M117 an it does it also after an reset... Weird. Anyone else have this?

**Comment by AlexMalfr** [comment]
🆙
I would love to be able to use M117 commands on my screen, especially in combination with Octoprint !

**Comment by SpankyDevil** [comment]
Has this feature been added yet?
